lib/percy/capybara.rb

```diff
@@ -42,6 +40,10 @@ def percy_snapshot(name, options = {})
     end
   end
 
+  private def current_page
+    Capybara.current_session
+  end
```

Comment:
We're concerned about the naming of this, but do we actually need it at all? For tiny methods like this I often ask myself if it's needed to being with. For example, this method is only called in the one place. Could we not have a local `page = Capybara.current_session` there instead of breaking it out into a dedicated method?

Reply:
Good call, I got wrapped up in my love for tiny ruby methods